jena-text/src/test/java/org/apache/jena/query/text/assembler/TestGenericAnalyzerAssembler.java

```diff
@@ -0,0 +1,198 @@
+package org.apache.jena.query.text.assembler;
```

Comment:
More FYI than comment:

If you want to run these test standalone, it needs to cause initialization with `static { JenaSystem.init() ; }` These test don't touch anything soon enough to cause that to happen implicitly.  These are not the only text tests that tis applies to.